invenio_rdm_records/records/stats/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2018 CERN.
+# Copyright (C) 2023 TU Wien.
+#
+# Invenio RDM Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Statistics integration for RDM records."""
+
+from .api import Statistics
+
+__all__ = Statistics
```

Comment:
```suggestion
__all__ = "Statistics"
```

Reply:
While I do agree, it is inconsistent accross modules:
- In some we have a list [] in some a set ()
- In some we quote, in the latest we don't


I can made the change you request, but black formatting removes it when there is only one item 🤷‍♂️

Reply:
nit, not a blocker